libs/dao-data/src/utils/proposalsStatus.ts

```diff
@@ -5,13 +5,15 @@ import {
 } from '@daohaus/common-utilities';
 import { QueryProposal } from '../types';
 
-export const isProposalUnsponsored = (proposal: QueryProposal): boolean =>
-  !proposal.sponsored &&
-  !proposal.cancelled &&
-  Number(proposal.expiration) >
-    Number(proposal.votingPeriod) +
-      Number(proposal.gracePeriod) +
-      nowInSeconds();
+export const isProposalUnsponsored = (proposal: QueryProposal): boolean => {
+  const notExpired =
+    Number(proposal.expiration) > 0 ||
+    Number(proposal.expiration) <
+      Number(proposal.votingPeriod) +
+        Number(proposal.gracePeriod) +
+        nowInSeconds();
+  return !proposal.sponsored && !proposal.cancelled && notExpired;
+};
```

Comment:
not that big of a deal but i think this unoptimizes the logic in this function -- checking the proposal first allows the other logic to be pushed to the end -- is there a specific reason this was adjusted?

could also do an if statement as well

```
if ( !proposal.sponsored && !proposal.cancelled) {
     Number(proposal.expiration) > 0 ||
    Number(proposal.expiration) <
      Number(proposal.votingPeriod) +
        Number(proposal.gracePeriod) +
        nowInSeconds();
} 

return false;
```